dig_test.go

```diff
@@ -2990,3 +2990,115 @@ func TestUnexportedFieldsFailures(t *testing.T) {
 			`bad argument 1: invalid value "foo" for "ignore-unexported" tag on field In: strconv.ParseBool: parsing "foo": invalid syntax`)
 	})
 }
+
+func TestConstructorInfoOption(t *testing.T) {
+	t.Parallel()
```

Comment:
optional: FYI, this only makes the top-level test parallel. If the subtests are parallel as well, you may add t.Parallel at the start of each.

nit: an empty line after t.Parallel call.